stix_shifter_modules/secretserver/stix_translation/json/event_from_stix_map.json

```diff
@@ -13,29 +13,31 @@
       ]
     }
   },
-
-  "user-account": {
+   "user-account": {
     "fields": {
-      "user_id": [
-        "UserId"
-      ]
+      "user_name": "username"
     }
   },
+  "email-addr": {
+        "fields": {
+            "value": "EmailAddress"
+        }
+    },
   "x-secret": {
     "fields": {
       "secret_name": ["SecretName"],
-      "user_name": ["username"],
+      "username": ["Unique_Identtification"],
       "secret_id": ["ItemId"],
-      "user_id": ["UserId"]
+      "user_id": ["UserId"],
```

Comment:
Why is this not just in the `user-account:user_id` object? No need to put it in a custom object.

Reply:
Done